.pre-commit-config.yaml

```diff
@@ -1,10 +1,6 @@
 repos:
--   repo: https://github.com/psf/black
-    rev: stable
+-   repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.3.7  # ruff version
```

Comment:
This comment is redundant, please drop it. (See https://github.com/mesh-adaptation/mesh-adaptation-docs/wiki/Development-Practices#code-comments.)

Reply:
Done in e037db64f41eee5ac7cfa541c09babcce8bf8679. Also removed the Black badge.